Soccar [UI]/src/nl/socnet/message/handler/StopGameMessageHandler.java

```diff
@@ -0,0 +1,30 @@
+package nl.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+import nl.socnet.message.StopGameMessage;
+
+/**
+ * Handler for StopGameMessage.
+ *
+ * @author PTS34A
+ */
+public class StopGameMessageHandler extends MessageHandler<StopGameMessage> {
+
+    @Override
+    protected void handle(Connection connection, StopGameMessage message) throws Exception {
+        //TODO : Exactly stop the game.

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)